doc/tutorials/basic/build_dashboard.md

```diff
@@ -0,0 +1,304 @@
+# Build a Dashboard
+
+In this tutorial we will build a beautiful dashboard showing key metrics of wind turbine manufacturers.
+
+<iframe src="https://awesome-panel-build-dashboard.hf.space" frameborder="0" style="width: 100%;height:1000px"></iframe>
```

Comment:
I've temporarily deployed the dashboard to my awesome-panel HF account.

Should be deployed to HF on Panel account. Or infrastructure used by Panel for apps.

Reply:
I've deployed to panel-Org on HF